docker/Dockerfile.package-cpu

```
@@ -1,6 +1,6 @@
 # Docker image: tlcpack/package-cpu
 
-FROM quay.io/pypa/manylinux2010_x86_64:2020-12-15-ba3529a
+FROM quay.io/pypa/manylinux2014_x86_64
```

These images are updated sometimes, it would be good to point to a specific tag, so that the build is reproducible, despite upstream image updates.

I collected the latest tag, to save you a trip to https://quay.io/repository/pypa/manylinux2014_x86_64?tab=tags

```suggestion
FROM quay.io/pypa/manylinux2014_x86_64:2021-04-27-6967be9
```